packages/nhsuk-frontend/src/nhsuk/components/contents-list/_index.scss

```
@@ -28,7 +28,7 @@
     display: block;
     left: 0;
     position: absolute;
-    top: 0.75em;
+    top: nhsuk-em(12, $nhsuk-base-font-size);
```

I think I prefer the units here, based on GOV.UK usage

```suggestion
    top: nhsuk-em(12px, $nhsuk-base-font-size);
```

<br>

Are we definitely sure `nhsuk-px-to-rem()` won't work?

---

Cool. Have added a line to the CHANGELOG too, happy to approve?